### PR TITLE
Add support to run GraalVM Check on pull request event trigger

### PR DESCRIPTION
**Pull request by TharmiganK**
> Related to : [`Add GraalVM check for the pull requests #4142`](https://github.com/ballerina-platform/ballerina-standard-library/issues/4142)

### CODE REVIEW

**Review by ThisaruGuruge — DISMISSED** [review]
LGTM

File: .github/workflows/build-with-bal-test-native-template.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ jobs:
     ubuntu-build-with-bal-test-native:
         name: Build with bal test native on Ubuntu
         runs-on: ubuntu-latest
+        if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check')
```
Comment (TharmiganK) [inline .github/workflows/build-with-bal-test-native-template.yml:38]:
```suggestion
        if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check') || github.event.pull_request.user.login != 'ballerina-bot'
```

File: .github/workflows/build-with-bal-test-native-template.yml [hunk 2]
```diff
@@ -97,6 +98,7 @@ jobs:
     windows-build-with-bal-test-native:
         name: Build with bal test native on Windows
         runs-on: windows-latest
+        if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check')
```
Comment (TharmiganK) [inline .github/workflows/build-with-bal-test-native-template.yml:101]:
```suggestion
        if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check') || github.event.pull_request.user.login != 'ballerina-bot'
```

**Review by ThisaruGuruge — APPROVED** [review]

**Comment by TharmiganK** [comment]
This has been not merged due to the U5 release process. Will review this after the U5 release.

**Pull request merged**